ush/test_retrieve_data.py

```diff
@@ -113,7 +113,7 @@ def test_gdas_ics_from_aws(self):
                 '--data_stores', 'aws',
                 '--external_model', 'GDAS',
                 '--fcst_hrs', '6', '9', '3',
-                '--output_path', out_path_tmpl,
+                '--output_path', tmp_dir,
```

Comment:
Here, it might be good to keep `out_path_tmpl` but to join it with `tmp_dir`...this is what I was thinking...

```
out_path_templ = os.path.join(tmp_dir, f"mem{{mem:03d}}")
```